fill/proto/interactive_filler.go

```diff
@@ -152,6 +152,7 @@ func (r *resolver) resolveField(f *desc.FieldDescriptor) error {
 			)
 			return msgr.resolve()
 		case descriptorpb.FieldDescriptorProto_TYPE_ENUM:
+			fmt.Printf("%s%s%s\n", r.color.ForegroundANSI(), f.GetName(), "\033[0m")
```

Comment:
Maybe, this line is not needed.
Instead, we can display a prefix string by passing it to [this call](https://github.com/ktr0731/evans/blob/fbe7341189e6f7d4f2bdc151e07fe9df9aab0414/fill/proto/interactive_filler.go#L126) (so we'll replace `? company.CompanyStatus:` by it), and [the prefix string](https://github.com/ktr0731/evans/blob/fbe7341189e6f7d4f2bdc151e07fe9df9aab0414/fill/proto/interactive_filler.go#L216) can be passed from [this call](https://github.com/ktr0731/evans/blob/fbe7341189e6f7d4f2bdc151e07fe9df9aab0414/fill/proto/interactive_filler.go#L155).

Reply:
Great, I finally understand the context. :bow: